easybuild/tools/run.py

```diff
@@ -226,6 +282,8 @@ def run_shell_cmd(cmd, fail_on_error=True, split_stderr=False, stdin=None, env=N
     :param qa_patterns: list of 2-tuples with patterns for questions + corresponding answers
     :param qa_wait_patterns: list of 2-tuples with patterns for non-questions
                              and number of iterations to allow these patterns to match with end out command output
```

Comment:
Is this correct? Specifically the 'and number of iterations to allow these patterns to match with end out command output' part.

Reply:
No it doesn't, but that does hint towards something in `run_cmd_qa` that some easyblocks may be using but is not supported yet in `run_shell_cmd`.
I can look into implementing that when we hit it, we should get this PR merged ASAP.

Fixed docstring in 204f3287b

Reply:
Looks like that feature isn't used in the current easyblocks that use the `no_qa` option of `run_cmd_qa`, so we can reimplement it if/when the need arises...